### PR TITLE
Dev 391 gjb ceip

### PR DESCRIPTION
**Pull request by GaryJBlake**

### CODE REVIEW

File: powerVCF.psm1 [hunk 1]
```diff
@@ -1806,3 +1806,104 @@ try {
...
+Export-ModuleMember -Function Get-VCFSettingCEIP
+
+Function Set-VCFSettingCEIP {
+
+<#
+    .SYNOPSIS
+    Sets the CEIP status (Enabled/Disabled) of the connected SDDC Manager
+	
+    .DESCRIPTION
+    The Set-VCFSettingCEIP cmdlet configuers the setting for CEIP of the connected SDDC Manager.
```
Comment (PowerVCF) [inline powerVCF.psm1]: Typo - configuers

Comment (GaryJBlake) [reply powerVCF.psm1]: Fixed

File: powerVCF.psm1 [hunk 2]
```diff
@@ -1806,3 +1806,104 @@ try {
...
+    .SYNOPSIS
+    Retrieves the setting for CEIP of the connected SDDC Manager
+	
+    .DESCRIPTION
+    The Get-VCFSettingCEIP cmdlet Retrieves the setting for CEIP of the connected SDDC Manager. 
+	
+    .EXAMPLE
+    PS C:\> This example shows how to get the current setting of CEIP
+	
+	PS C:\> Get-VCFSettingCEIPs
```
Comment (PowerVCF) [inline powerVCF.psm1]: Typo - additional s

File: powerVCF.psm1 [hunk 3]
```diff
@@ -1806,3 +1806,104 @@ try {
...
+    .SYNOPSIS
+    Sets the CEIP status (Enabled/Disabled) of the connected SDDC Manager
+	
+    .DESCRIPTION
+    The Set-VCFSettingCEIP cmdlet configuers the setting for CEIP of the connected SDDC Manager. 
+	
+    .EXAMPLE
+    PS C:\> This example shows how to disable CEIP of the connected SDDC Manager
+	
+    PS C:\> Set-VCFSettingCEIP ENABLE
```
Comment (PowerVCF) [inline powerVCF.psm1]: Do you not need to pass -ceipSetting? That would align it to other functions that accept multiple params

File: powerVCF.psm1 (hunk 2 above)
Comment (GaryJBlake) [reply powerVCF.psm1]: Fixed

File: powerVCF.psm1 (hunk 3 above)
Comment (GaryJBlake) [reply powerVCF.psm1]: Added to example

**Comment by PowerVCF** [comment]
Nice work!

**Pull request merged**